src/main/java/frc/robot/subsystems/IndexSubsystem.java

```diff
@@ -24,9 +24,18 @@
   // Motors
   
   private TalonSRX tower = new TalonSRX(Constants.TOWER_INDEX_ID);
+  private DigitalInput bannerSensor1 = new DigitalInput(Constants.DIO_PORT_0);
+  private DigitalInput bannerSensor2 = new DigitalInput(Constants.DIO_PORT_1);
 
-  private BannerSensorSubsystem bannerSensorControl= new BannerSensorSubsystem();
-  
+
+  public enum STATES {
```

Comment:
Why are there two `STATES` `enum`s in the repository? There should only be one.

Also, `STATES` should be `States`.

Reply:
Is `STATES` even being used anywhere to begin with?